CONTRIBUTING.md

```diff
@@ -15,11 +15,13 @@ In this topic, we'll cover:
 
 ## Process for contributing
 
-**Step 1:** Open an issue describing the article you wish to write and how it relates to existing content.
+**Step 1:** Open an [issue](https://github.com/dotnet/core-docs/issues) describing the article you wish to write and how it relates to existing content.
 The content inside the **docs** folder is organized into sections that are reflected in the Table of Contents (TOC). Define where the topic will be located in the TOC. Get feedback on your proposal. 
 
 You can skip this step for small changes.
 
+You can also look at our [open issues](https://github.com/dotnet/core-docs/issues) list and volunteer to work on the ones you're interested in. We use the [up-for-grabs](https://github.com/dotnet/core-docs/labels/up-for-grabs) label to tag issues we think the community could easily help us out.
```

Comment:
Changed the sentence based on your suggestions. Thanks! 

Now for soliciting reviewers I think we need a more drastic change to this guide. I'd actually reorder the topic to have the following sections (just a suggestion):
- Contributing to content
- Contributing to samples
- Reviewing PRs
- DO's and DON'Ts
- Building the docs
- Contributor License Agreement (I just noticed this content went missing, I'll add it back)

Thoughts?